specs/MultiProfile.md

```diff
@@ -217,6 +232,80 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(
 }
 ```
 
+### Manage password-autosave and general-autofill settings in profile
+
+```cpp
+HRESULT AppWindow::ManagePasswordAutosaveInProfile(ICoreWebView2Controller* controller)
```

Comment:
nit, maybe call these sample methods "Toggle*" (instead of "Manage*") so it's clear we're blindly flipping the bit instead of consuming the current value.

Reply:
Good idea, done.